index.bs

```diff
@@ -2898,3 +2903,7 @@ To decode a PSK `P` given a QR code, follow these steps:
 
 1. Obtain the string `N` by decoding the QR code.
 2. Parse `N` as a hexadecimal number to obtain `P`.
+
+Appendix D: Entire Flow Chart {#appendix-d}
+=============================
+<img src="./images/entire_flow_chart.svg" alt="Entire flow chart" style="width: 100%; height: 100%">
```

Comment:
It seems a good idea to flag the section as informative to clarify that, in case of discrepancy, the normative prose takes precedence. Bikeshed does not have a magic keyword to do that, so section needs to be wrapped in a `<section>` and text added explicitly.

I also propose to provide a more complete `alt` text that describes the image to meet [HTML requirements](https://html.spec.whatwg.org/multipage/images.html#alt).

Two other changes:
- I'm not sure what `height: 100%` changes. I dropped it. Feel free to reintroduce if I missed something!
- The `no-autosize` attribute tells Bikeshed not to try to compute the dimensions of the image (which it cannot because it's an SVG file).

```suggestion
<section class="informative">
Appendix D: Entire Flow Chart {#appendix-d}
=============================
*This section is non-normative.*

<img no-autosize src="./images/entire_flow_chart.svg" alt="Before a listening agent (client) and an advertising agent (server) may exchange application messages, they first need to discover each other through an mDNS exchange, establish a QUIC connection through ClientHello and ServerHello exchanges and sharing of certificates, and run an SPAKE2 authentication handshake and confirmation. They may also exchange messages to discover metadata at any time after a QUIC connection has been established." style="width: 100%">
</section>
```

Reply:
Thanks @tidoust! @wangw-1991 to integrate and @mfoltzgoogle to review.

Reply:
Thanks @tidoust for the suggestions. Can you help merge this?